src/components/table/mixins/sort.js

```diff
@@ -19,43 +19,53 @@ export default {
       if (!this.isSortable || !this.sortBy) {
         return
       }
+      if (this.sortBy === "name") {
+        let folders = this.data.filter(i => i.type === "folder")
```

Comment:
I'd like to see filtering and sorting chained and (because it's not modified after that) stored in a `const`:
`const folders = this.data.filter(i => i.type === "folder").sort((a, b) => this.sortData(a, b))`

Same for files.

Reply:
yes good improvement